src/openagi/llms/groq.py

```diff
@@ -46,10 +41,22 @@ def run(self, input_data: str):
             self.load()
         if not self.llm:
             raise ValueError("`llm` attribute not set.")
-        message = HumanMessage(content=input_data)
-        resp = self.llm([message])
-        return resp.content
-    
+        try:
+            chat_completion = self.llm.chat.completions.create(
```

Comment:
Actually Langchain internally have Human messages which has the system prompt. Would you add an system prompt as well.